frog/imports/api/activities.js

```diff
@@ -118,15 +129,20 @@ export const copyOperatorIntoGraphOperator = (
   });
 };
 
-export const removeGraph = graphId => Meteor.call('graph.flush.all', graphId);
+export const removeGraph = (graphId: string) =>
+  Meteor.call('graph.flush.all', graphId);
 
 export const deleteDatabase = () => Meteor.call('graph.flush.db');
 
-export const dragGraphActivity = (id, xPosition) => {
+export const dragGraphActivity = (id: string, xPosition: number) => {
```

Comment:
This function is not used anywhere anymore, should be removed (I guess it's from our old implementation of graph editor)